packages/integrations/netlify/src/integration-edge-functions.ts

```diff
@@ -11,6 +11,16 @@ export function getAdapter(): AstroAdapter {
 		name: '@astrojs/netlify/edge-functions',
 		serverEntrypoint: '@astrojs/netlify/netlify-edge-functions.js',
 		exports: ['default'],
+		supportedAstroFeatures: {
+			hybridOutput: 'Stable',
+			staticOutput: 'Stable',
+			serverOutput: 'Stable',
+			assets: {
+				supportKind: 'Unsupported',
```

Comment:
```suggestion
				supportKind: 'Stable',
```

Same as Cloudflare, every adapter supports `assets`, it's the services they don't support